tests/ethercat/test_pdo.py

```diff
@@ -239,6 +244,7 @@ def test_servo_reset_pdos(connect_to_slave, create_pdo_map):
 
 
 @pytest.mark.ethercat
+@pytest.mark.skip
```

Comment:
Why do you skip?

Reply:
It was mistake, I removed the mark: [27405b8](https://github.com/ingeniamc/ingenialink-python/pull/370/commits/27405b8a8349395b616558d793141ac57626fa44)